modin/pandas/test/internals/test_repartition.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+
+import modin.pandas as pd
+from modin.config import NPartitions
+
+NPartitions.put(4)
```

Comment:
Why is this needed?

Reply:
We use this for almost all tests.

Reply:
I wonder why?

Reply:
I don't know for sure, but otherwise only 2 cores would be used. Apparently this case is not interesting.

Reply:
makes sense